engineyard/ey-cookbooks-dev-v7
# [FBZ-10419] - Migrate Resque Recipe

**Description of your patch**

Adds support for Resque by migrating recipe from v6 to v7

**Recommended Release Notes**

Resque support added

**Estimated risk**

Moderate risk only involves own components

**Components involved**

* cookbooks/ey-resque
* custom-cookbooks/resque

**Dependencies**

None

**Description of testing done**

* Booted up fresh Environment
* Added utility instance for resque
* Updated environment variables to enable resque
* Uploaded custom recipe to include resque in chef run
* Triggered chef run
* Verified resque workers

**QA Instructions**

* Check README.md in resque recipe
* Create environment with resque enabled
* Create custom recipe to include resque in chef run
* Apply changes on the environment
* Verify if workers are running
